core/vm/operations_acl.go

```diff
@@ -103,14 +110,23 @@ func makeGasSStoreFunc(clearingRefund uint64) gasFunc {
 func gasSLoadEIP2929(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	loc := stack.peek()
 	slot := common.Hash(loc.Bytes32())
+	var gasUsed uint64
+
+	if evm.chainConfig.IsCancun(evm.Context.BlockNumber) {
+		where := stack.Back(0)
+		addr := contract.Address()
+		index := trieUtils.GetTreeKeyStorageSlot(addr[:], where)
+		gasUsed += evm.Accesses.TouchAddressOnReadAndComputeGas(index)
+	}
```

Comment:
:thinking: this change is a bit weird because access lists are supposed to be removed with the verkle tree change, and yet we are charging gas for this... If anything, this funciton should simply not be called in a verkle tree context.